src/activities/categorization/Categorization.qml

```diff
@@ -36,6 +36,7 @@ ActivityBase {
 
     property string boardsUrl: "qrc:/gcompris/src/activities/categorization/resource/"
     property bool vert: background.width < background.height
+    property string datasetURL: "../../GCompris-qt/src/activities/categorization/resource/board/"
```

Comment:
Using opendir is a bad choice here: it's not portable and it won't work once the software is installed. ../GCompris-qt works for you because the source code of gcompris is on this path/ Once installed, the source code is not installed so you no more have access to this path.